### PR TITLE
Add missing DOMCharacterData::appendData() return true description

### PR DESCRIPTION
**Pull request by othercorey**
The return type is bool not void and always returns true.

### CODE REVIEW

**Comment by othercorey** [comment]
@cmb69 Can I poke you to finish these last PRs?

File: reference/dom/domcharacterdata/appenddata.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   &return.void;
+   Always returns &true;.
```
Comment (Girgias) [inline reference/dom/domcharacterdata/appenddata.xml:38]:
```suggestion
   &return.true.always;
```

**Comment by Girgias** [comment]
If this is the case the stubs in php-src should also be updated to reflect this.

**Comment by othercorey** [comment]
The stub is type `bool`. Wasn't sure if everything currently returning `true` should be `true` in the stubs.

**Comment by Girgias** [comment]
> The stub is type `bool`. Wasn't sure if everything currently returning `true` should be `true` in the stubs.

IMHO it should, do you want to make the PR to php-src?

**Comment by tstarling** [comment]
Guess I should have rebased that manually. Apparently the change had already been made, and the commit was a no-op.

**Comment by Girgias** [comment]
Ah yeah, must have gone through in one of the other stubs syncs.

**Pull request merged**